packages/loader/container-definitions/src/deltas.ts

```diff
@@ -157,6 +157,7 @@ export interface IDeltaManager<T, U> extends IEventProvider<IDeltaManagerEvents>
      * and do not know if user has write access to a file.
      */
     readonly readonly?: boolean;
+    readonly storageOnly: boolean | undefined;
```

Comment:
can you also add this to this readme where we cover readonly states: fluid\packages\loader\container-loader\README.md